README.md

```diff
@@ -10,17 +10,27 @@
 
 ## What is SimBricks?
 
-SimBricks is a simulator framework aiming to enable true end-to-end simulation
-of modern data center network systems, including multiple servers running a full
-software stack with unmodified OS and applications, network topologies and
-devices, as well as other off the shelf and custom hardware components. Instead
-of designing a new simulator from scratch, SimBricks combines and connects
-multiple existing simulators for different components into a simulated full
-system. Our primary use-case for SimBricks is computer systems, networks, and
-architecture research. Our [paper](https://arxiv.org/abs/2012.14219) provides a
-more detailed discussion of technical details and use-cases.
-
-Currently, SimBricks includes the following simulators:
+SimBricks is an open-source simulation framework for intricate HW-SW systems
+that enables rapid virtual prototyping and meaningful end-to-end evaluation in
+simulation. SimBricks modularly combines and connects battle-tested simulators
+for different components: machines (e.g. QEMU, gem5, Simics), hardware
```

Comment:
I think we shoul not list more or less all supported simulators by name here. For that we have the list below.

Reply:
My reasoning for that is that you immediately get an idea about what we mean by "machine" or "hardware".